test/blackbox-tests/test-cases/dune-project-meta/basic-generate.t

```diff
@@ -4,22 +4,23 @@ Simple test
 The `dune build` should generate the opam file
 
   $ cat >dune-project <<EOF
-  > (lang dune 1.10)
+  > (lang dune 3.18)
```

Comment:
I don't think this is a good change, we should still retain tests that check that 1.10 era projects are supported. It's better to add an additional test for the maintenance intent (and that will also cause less changes in the cram tests).

Reply:
Agree!